src/manager/node.c

```diff
@@ -1082,6 +1091,13 @@ static int agent_request_callback(sd_bus_message *m, void *userdata, UNUSED sd_b
         return req->cb(req, m, ret_error);
 }
 
+int agent_request_cancel(AgentRequest *r) {
+        _cleanup_agent_request_ AgentRequest *req = r;
```

Comment:
Just for confirmation: 
This `_cleanup_agent_request_` does an `agent_request_unref` and the implied `node_unref`, right?

Reply:
Yes, correct